### PR TITLE
Remove uid/gid mapping limit depend on kernel

### PR DESCRIPTION
**Pull request by hqhq**
Signed-off-by: Qiang Huang <h.huangqiang@huawei.com>

### CODE REVIEW

File: config-linux.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ Each entry has the following structure:
 * **`size`** (uint32, REQUIRED)* - is the number of ids to be mapped.
 
 The runtime SHOULD NOT modify the ownership of referenced filesystems to realize the mapping.
-There is a limit of 5 mappings which is the Linux kernel hard limit.
+Note that there might be a limit of mapping entries by the [kernel][user-namespaces].
```
Comment (mrunalp) [inline config-linux.md]: nit: ... in the Linux kernel.

**Comment by crosbymichael** [comment]
LGTM

File: config-linux.md (hunk 1 above)
Comment (mikebrow) [inline config-linux.md]: The number of mapping entries MAY be limited by the [kernel][user-namespaces].

**Comment by hqhq** [comment]
Updated.

**Comment by mrunalp** [comment]
LGTM

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**